.asf.yaml

```diff
@@ -34,7 +34,7 @@ github:
     rebase: true
 
   protected_branches:
-    main:
+    1.8.x:
```

Comment:
wouldn't we need to update this for all branches then (which I don't think we would retroactively want to do)?

Reply:
On the other hand, we need to remember update this for future release branches.

Reply:
> On the other hand, we need to remember updating this for future release branches.

yes and that's my main concern. I think if somebody accidentally merges something, then we can easily revert in that case.

Reply:
I don't think this change makes such sense: `.asf.yaml` is parsed only on `main` branch.
And also, for "release flexibility", I'm not sure we should protect release branches.

Reply:
Okay, let me open a PR against `main` branch. I still think release branch needs to be protected since it could be maintained for a long time

Reply:
we typically tell users to always upgrade to the latest version, so at max we're maintaining the last released minor version branch and we don't do any new patch releases if there's a newer minor version